eZ/Publish/API/Repository/SearchService.php

```diff
@@ -96,4 +97,13 @@ public function suggest($prefix, $fieldPaths = array(), $limit = 10, Criterion $
      * @return \eZ\Publish\API\Repository\Values\Content\Search\SearchResult
      */
     public function findLocations(LocationQuery $query, array $languageFilter = array(), $filterOnUserPermissions = true);
+
+    /**
+     * Get SortClause built from $location's sort options.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Query\SortClause
```

Comment:
Thinking it should perhaps return `SortClause[]` here, just in case Repository changes in the future to allow several sort fields on locations for fallback.